setup.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 3YOURMIND GmbH
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from codecs import open
+from os import path
+from setuptools import setup, find_packages
+
+
+PROJECT_DIR = path.abspath(path.dirname(__file__))
+
+with open(path.join(PROJECT_DIR, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+install_requirements = [
+]
+
+test_requirements = [
+    'pytest',
+    'django',
+    'django-fake-database-backends'
+]
+
+setup(
+    name='django-migration-linter',
+    version='0.0.1',
+
+    description='Fetect backward incompatible migrations for your django project',
+    long_description=long_description,
+    url='https://github.com/3YOURMIND/django-migration-linter',
+    author='3YOURMIND GmbH',
+    license='Apache License 2.0',
+
+    packages=find_packages(exclude=['tests/']),
+    install_requires=install_requirements,
+    extras_require={
+        'test': test_requirements
+    },
+
+    keywords='django migration lint linter database backward compatibility',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'Environment :: Web Environment',
+        'Framework :: Django',
+
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        # TODO verify Python 3 compatible
```

Comment:
should this stay here?

Reply:
You're right, I removed the comments.

I will anyway tackle the Python 3 compatibility issue soon.